compiler/main/driver.cpp

```diff
@@ -721,6 +724,7 @@ static ArgumentDescription arg_desc[] = {
  {"local-checks", ' ', NULL, "Enable [disable] local block checking", "n", &fNoLocalChecks, NULL, NULL},
  {"nil-checks", ' ', NULL, "Enable [disable] nil checking", "n", &fNoNilChecks, "CHPL_NO_NIL_CHECKS", NULL},
  {"stack-checks", ' ', NULL, "Enable [disable] stack overflow checking", "n", &fNoStackChecks, "CHPL_STACK_CHECKS", handleStackCheck},
+ {"cast-checks", ' ', NULL, "Enable [disable] safe cast checking", "n", &fNoCastChecks, NULL, NULL},
```

Comment:
I think this and the man page need a clearer explanation. As it is, I would assume this meant safe cast checking on all casts, not just ones inside of the safe_cast function.